.github/workflows/lock.yml

```
@@ -0,0 +1,18 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
```

```suggestion
      - uses: dessant/lock-threads@ 1621939cecf8586399a6b60d2a7af9469232b5b6
```

let's pin specific versions so Dependabot tells us about updates